apps/tangle-dapp/components/ServicesKeyMetricItem/ServicesKeyMetricItem.tsx

```diff
@@ -0,0 +1,75 @@
+import { ArrowRight } from '@webb-tools/icons';
+import { SkeletonLoader, Typography } from '@webb-tools/webb-ui-components';
+import { getRoundedAmountString } from '@webb-tools/webb-ui-components/utils';
+import cx from 'classnames';
+import { FC } from 'react';
+
+import { ServicesKeyMetricItemProps } from './types';
+
+const ServicesKeyMetricItem: FC<ServicesKeyMetricItemProps> = ({
+  title,
+  Icon,
+  isLoading,
+  isError,
+  value,
+  changeRate,
+  className,
+}) => {
+  return (
+    <div
+      className={cx(
```

Comment:
I'm not sure if cx will handle Tailwind class name conflicts, I think we should be using twMerge here.